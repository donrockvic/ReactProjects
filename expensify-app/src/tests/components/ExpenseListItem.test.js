import React from 'react';
import ReactShallowRender from 'react-test-renderer/shallow';
import ExpenseListItem from '../../components/ExpenseListItem';
import expences from '../fixtuers/expenses';

test('should render ExpenseList with Item', () => {
    const renderer = new ReactShallowRender();
    renderer.render(<ExpenseListItem {...expences[1]} />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});