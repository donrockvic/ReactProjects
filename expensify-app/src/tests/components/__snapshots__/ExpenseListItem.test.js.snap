// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with Item 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
       
      Rent
       
    </h3>
  </Link>
  <p>
    20000
    -
    -345600000
  </p>
</div>
`;
